test/Neumark.js

```diff
@@ -13,10 +11,6 @@ contract("Neumark", accounts => {
     await neumark.changeController(controller.address);
   });
 
-  it("should deploy", async () => {
-    console.log(`\tNeumark took ${gasCost(neumark)}.`);
-    console.log(`\tController took ${gasCost(controller)}.`);
-  });
```

Comment:
Gas cost of contract deployment is also an important resource. It is possible to write contracts that are to big to deploy.